configs/ote/face-detection/face-detection-0200/template.yaml

```diff
@@ -0,0 +1,101 @@
+# Description.
+name: face-detection-0200
+task_type: DETECTION
+task_family: VISION
+instantiation: CLASS
+summary: Face Detection based on MobileNetV2 (SSD).
+
+# Algo backend.
+# Would be better to have something more precise and self-contained, e.g.:  docker image or path to Python venv activation script?
+framework: OTEDetection v2.9.1
+
+dependencies:
+- source: ../../../../mmdet/apis/ote/apis/detection/configuration.yaml
+  destination: configuration.yaml
```

Comment:
`dependencies` is deprecated.

Reply:
I know, did not merge changes for the models yet

Reply:
These files are removed from the current PR.